lms/djangoapps/teams/static/teams/js/spec/edit_team_spec.js

```diff
@@ -0,0 +1,128 @@
+define([
+    'jquery',
+    'underscore',
+    'backbone',
+    'common/js/spec_helpers/ajax_helpers',
+    'teams/js/views/edit_team'
+], function ($, _, Backbone, AjaxHelpers, TeamEditView) {
+    'use strict';
+
+    describe('EditTeam', function () {
+        var teamEditView,
+            verifyValidation = function (fieldsData) {
+                _.each(fieldsData, function (fieldData) {
+                    teamEditView.$(fieldData[0]).val(fieldData[1]);
+                });
+                teamEditView.$('.create-team.form-actions .action-primary').click();
+                expect(teamEditView.$('.wrapper-msg').is(':visible')).toBeTruthy();
+                expect(teamEditView.$('.wrapper-msg .title').text().trim()).toBe("Oops!");
```

Comment:
This method should check for particular validation messages on the fields as well as whatever class makes the fields red; not just the "Oops!" message.